### PR TITLE
Routine rubocop fixes

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description

After #4632 was merged, I re-ran rubocop against the code

## Related Tickets & Documents

#4632

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -26,11 +26,16 @@ RSpec/ExampleLength:
...
-# Offense count: 427
+# Offense count: 444
 # Configuration parameters: AggregateFailuresByDefault.
 RSpec/MultipleExpectations:
   Max: 8
 
+# Offense count: 1
+RSpec/UnspecifiedException:
+  Exclude:
+    - 'spec/requests/messages_spec.rb'
```
Comment (rhymes) [inline .rubocop_todo.yml]: this is related to this PR attempt, https://github.com/thepracticaldev/dev.to/pull/4573

until we fix the root of the problem, i'm +1 on keeping this in the todo

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -562,58 +562,60 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
     end
   end
 
-  it "creates proper body class with defaults" do
-    expect(user.decorate.config_body_class).to eq("default default-article-body pro-status-#{user.pro?} trusted-status-#{user.trusted} #{user.config_navbar}-navbar-config")
-  end
+  describe "theming properties" do
```
Comment (rhymes) [inline spec/models/user_spec.rb]: grouped these

File: app/models/user.rb [hunk 3]
```diff
@@ -106,29 +106,29 @@ class User < ApplicationRecord
             format: /\A(http(s)?:\/\/)?(www.twitch.tv|twitch.tv)\/.*\Z/
   validates :shirt_gender,
             inclusion: { in: %w[unisex womens],
-                         message: "%{value} is not a valid shirt style" },
+                         message: "%<value>s is not a valid shirt style" },
```
Comment (mstruve) [inline app/models/user.rb]: I have never seen this syntax before, do you have a link to the docs for it? I was coming up empty-handed trying to google.

Comment (rhymes) [reply app/models/user.rb]: I got it from [rubocop's Style/FormatStringToken](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/FormatStringToken) and from the [Ruby Style Guide's 
Named Format Tokens rule](https://github.com/rubocop-hq/ruby-style-guide#named-format-tokens).

I think they both refer to [Kernel#format](https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-format)'s advanced examples:

```text
For more complex formatting, Ruby supports a reference by name. %<name>s style uses format style, but %{name} style doesn't.

Examples:

sprintf("%<foo>d : %<bar>f", { :foo => 1, :bar => 2 })
  #=> 1 : 2.000000
sprintf("%{foo}f", { :foo => 1 })
  # => "1f"
```

Basically the reason is to embed the value type in the format string

Comment (mstruve) [reply app/models/user.rb]: TIL, thanks!

**Pull request merged**